corehq/toggles.py

```diff
@@ -731,7 +731,7 @@ def _commtrackify(domain_name, toggle_is_enabled):
     'Use New CloudCare',
     TAG_PRODUCT_PATH,
     [NAMESPACE_DOMAIN],
-    randomness=0.7,
+    randomness=1.0,
```

Comment:
ah, i'll do that. my thinking was it'd be easy to turn down, but i guess it still takes a revert and deploy. would be sweet if we could turn the randomness up in the ui...